static/app/views/eventsV2/table/columnEditModal.spec.jsx

```diff
@@ -20,13 +24,45 @@ function mountModal({columns, onApply, customMeasurements}, initialData) {
       closeModal={() => void 0}
       customMeasurements={customMeasurements}
     />,
-    initialData.routerContext
+    {context: initialData.routerContext}
   );
 }
 
-describe('EventsV2 -> ColumnEditModal', function () {
-  enforceActOnUseLegacyStoreHook();
+const findAllQueryFields = () => screen.findAllByTestId('queryField');
```

Comment:
this is a little difficult to understand

Reply:
Just going to add a comment here for now.